### PR TITLE
Create delegate method for changes in tokenFieldHeight

### PR DESCRIPTION
**Pull request by dasmer**
This way, implementations that use auto-layout can respond to height changes by modifying height constraint constants.

### CODE REVIEW

File: VENTokenField/VENTokenField.m [hunk 1]
```diff
@@ -375,19 +375,25 @@ - (UILabel *)toLabel
...
+            height = currentY + [self heightForToken] + self.verticalInset * 2;
         } else {
-            [self setHeight:self.originalHeight];
+            height = self.originalHeight;
         }
     }
+    if (oldHeight != height
+        && [self.delegate respondsToSelector:@selector(tokenField:didChangeContentHeight:)]) {
+        [self.delegate tokenField:self didChangeContentHeight:height];
+    }
```
Comment (ayanonagon) [inline VENTokenField/VENTokenField.m]: Do we need to also call `[self setHeight:]` when there is no delegate set for backwards compatibility?

Comment (dasmer) [inline VENTokenField/VENTokenField.m]: goood point

Comment (dasmer) [inline VENTokenField/VENTokenField.m]: thanks for that catch @ayanonagon 
done in a6c26aac8cf504ba56610558a370f5fa44eaa998

**Comment by ayanonagon** [comment]
:green_apple: Thanks for making the change!

**Pull request merged**